recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
   build:
     - cmake
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
Why is C++ needed? IIUC METIS is just C. Is there something I'm missing?

Reply:
One of the builds on the master branch fails with:

```
  No CMAKE_CXX_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.
```

Reply:
So the reason is that `cmake` *always* looks for a C++ compiler, even for pure C project, unless the cmake project specifies otherwise, by stating `project (METIS C)` instead of `project(METIS)`.

Hence this is the proper fix.

Reply:
If that's true, then there should probably be:

```yaml
build:
  ignore_run_exports:
    - {{ compiler('cxx') }}
```

to avoid pulling in the libc++ runtime lib dependencies

Reply:
Thanks @minrk I just added a run exports session and a comment about the `compiler('cxx')` dependency.

Reply:
Makes sense. Thanks @SylvainCorlay. Also thanks for the `ignore_run_exports` fix, @minrk.